source/unified-test-format/tests/invalid/entity-client-observeTracingMessages-additionalProperties.yml

```
@@ -0,0 +1,14 @@
+description: "entity-client-observeTracingMessages-additionalProperties"
+
+schemaVersion: "1.26"
+
+createEntities:
+  - client:
+      id: &client0 "client0"
+      observeTracingMessages: {
+        foo: "bar"
+      }
```

```suggestion
      observeTracingMessages:
        foo: "bar"
```

The braces are redundant in YAML if you're already using multi-line syntax.